src/networking.c

```diff
@@ -1951,6 +1955,12 @@ int processMultibulkBuffer(client *c) {
             /* Not enough data (+2 == trailing \r\n) */
             break;
         } else {
+            /* Check if we have space in argv, grow if needed */
+            if (c->argc >= c->argv_len) {
+                c->argv_len *= 2;
```

Comment:
```suggestion
                c->argv_len = min(c->argv_len*2, c->multibulklen);
```

Reply:
Good point, but the suggestion won't work because `multibulklen` is decremented in the arg parsing loop. It should be
```c
min(c->argv_len*2, c->multibulklen+c->argc);
```

Reply:
fixed